docs/components/movement-sensor/gps/gps-rtk.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Configure an NTRIP-based RTK GPS"
+linkTitle: "gps-rtk"
+weight: 10
+type: "docs"
+description: "Configure an NTRIP-based RTK GPS."
+# SMEs: Rand
+---
+
+{{% alert title="Note" color="note" %}}
+
+The `gps-rtk` model is an experimental feature.
+Stability is not guaranteed.
+Breaking changes are likely to occur, and occur often.
+
+{{% /alert %}}
+
+A global positioning system (GPS) receives signals from satellites in the earth’s orbit to determine where it is and how fast it is going.
+All supported GPS models provide date for the `Position`, `CompassHeading` and `LinearVelocity` methods.
+Fix and Correction data are available by using the sensor `GetReadings` method, which is available because GPSes wrap the [sensor component](../../../sensor/).

```

Comment:
```suggestion
You can obtain fix and correction data by using the sensor `GetReadings` method, which is available because GPSes wrap the [sensor component](../../../sensor/).
```